src/unix/qnx.c

```diff
@@ -113,15 +147,18 @@ int uv_resident_set_memory(size_t* rss) {
 
 
 int uv_uptime(double* uptime) {
-  struct qtime_entry* qtime = _SYSPAGE_ENTRY(_syspage_ptr, qtime);
-  *uptime = (qtime->nsec / 1000000000.0);
+  struct timespec ts;
+  int ret = clock_gettime(CLOCK_MONOTONIC, &ts);
+  if (ret == -1)
+    return ret;
```

Comment:
```suggestion

  if (clock_gettime(CLOCK_MONOTONIC, &ts))
    return UV__ERR(errno);
```
Otherwise callers see a nonsensical UV_EINVAL (the error that's usually mapped to -1)

Reply:
You're right, I completely missed this, thank you for highlighting